data/Dockerfiles/letsencrypt/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+
+RUN apk add --update \
+    bash \
+    curl \
+    openssl \
+    && rm -rf /var/cache/apk/*
+
+RUN curl -L https://github.com/lukas2511/dehydrated/archive/master.tar.gz \
```

Comment:
Using multiple RUNs adds additional layers, by using && the work is done in one.
`&& curl -L https://github.com/lukas2511/dehydrated/archive/master.tar.gz`

It is also prefered to lock the external depencendy to a specific version:
https://github.com/lukas2511/dehydrated/archive/v0.4.0.zip

The container philosophy is to create an immutable application.

An alternative could also be to use one of the ACME clients that have already been packaged for Alpine:
https://pkgs.alpinelinux.org/package/v3.6/community/x86_64/acme-client (in C)
https://pkgs.alpinelinux.org/package/edge/community/x86_64/lego (in GO)